doc/sections/module.md

```diff
@@ -1,45 +1,84 @@
 Module system
 -------------
 
-Lambdapi has a (very) light module system based on file paths. Although it has
-several limitations,  the module system allows the splitting developments into
-several files, and even several folders. The main thing to know is that a file
-holds a single module, which name corresponds to that of the file. For example
-a file `nat.lp` in the working directory will define the `nat` module,  and it
-can be referred as such in other files (or modules).
-
-However, note that `nat.lp` only defines module `nat` if `lambdapi` is invoked
-in the directory containing the file. Now, if the current directory contains a
-directory `lib`, itself containing a file `boolean.lp`, then the corresponding
-module is accessed using a full qualification `lib.boolean`. To illustrate the
-behaviour of the system, consider the following example.
-```bash
-working_directory  # lambdapi is invoked in that directory
-├── main.lp        # this module must always be refered as "main"
-├── bool.lp        # this module must always be refered as "bool"
-├── church
-│   ├── nat.lp     # this module must always be refered as "church.nat"
-│   └── list.lp    # this module must always be refered as "church.list"
-└── scott
-    ├── nat.lp     # this module must always be refered as "scott.nat"
-    └── list.lp    # this module must always be refered as "scott.list"
+Lambdapi has a (very) light module system based on file paths. It allows us to
+split our developments into several files, and even several folders.  The main
+thing to know is that a file holds a single module,  and accessing this module
+in other files requires using its *full* module path.
+
+#### Module path
+
+The module path that corresponds to a source file is defined using the name of
+the file, but also the position of the file under the *library root* (a folder
+under which the standard library and all packages are installed).
+
```

Comment:
I would prefer not to speak about some standard library as we have none. And the day we will have one, if any, it should be in a distinct repo anyway, like it is done with Lean or Agda.

Reply:
Well, this is intended as an example of package creation. I don't think it is harmful, and we can actually use if for experimenting with things. It can always be made into a separate repository later, but I don't think this is a good idea for such a small tool.